paparazzi-annotations/api/paparazzi-annotations.api

```diff
@@ -1,3 +1,39 @@
 public abstract interface annotation class app/cash/paparazzi/annotations/Paparazzi : java/lang/annotation/Annotation {
 }
 
+public abstract interface class app/cash/paparazzi/annotations/PaparazziPreviewData {
+}
+
+public final class app/cash/paparazzi/annotations/PaparazziPreviewData$Default : app/cash/paparazzi/annotations/PaparazziPreviewData {
```

Comment:
this is implementation detail right?  so this shouldn't be exposed as public api

Reply:
thanks.  now this PR is a bunch of dependency version bumps, can this be moved (maybe even split if multiple concerns are being bundled) to its/their respective PR(s)?

Reply:
no dep bump is in this PR, just defined the version as a variable

Reply:
oh gotcha, missed that, my bad.  in that case, let's change the commit title and PR title appropriately, i.e., "Align compose dependencies to same version"?

Reply:
I see, yes this pr is just those updates, so i changed them appropriately

Reply:
I see you updated the PR title but I don't think that will update the commit title when merging into history.  Also, nit: cleanup is a bit vague.